addons_marketplace/marketplace-manager/services/marketplace_service.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import threading
+
+import docker
+from db import marketplace_db
+
+
+def verify_addon(addon_id, addon):
+    logging.info(f"verifying addon-{addon_id}...")
+    client = docker.from_env()
+    for service in addon["services"]:
+        image_uri = service.get("image_uri")
+        try:
```

Comment:
Do I understand this correctly?:
The verification is checking if the image can be pulled?

Reply:
yes! inefficient but should be enough for now. I am working on a go implementation that does this more effieciently.